jobs/MasterCI/MasterCI

```diff
@@ -1,42 +1,60 @@
 node{
-    deleteDir()
-    checkout scm
-    def shareMethod = load("jobs/ShareMethod.groovy")
-    try{
-        stage("Create Manifest"){
-            load("jobs/create_manifest/create_manifest.groovy")
-        }
-        def manifest_name=env.MANIFEST_FILE_URL.tokenize('/')[-1];
-        currentBuild.description = "<a href=${env.MANIFEST_FILE_URL}>${manifest_name}</a>";
+    withEnv([
+        "branch=${env.branch}",
+        "IS_OFFICIAL_RELEASE=false",
+        "date=current",
```

Comment:
Yes, I will remove "date"/"timezone" from Jenkins build parameter after the PR merged.

The benefit of ENV Variable here:
1. Set value for some ENV Variable, such as ```date```
2. Decouple pipeline script with Jenkins server.
     It's easy for us to run the pipeline in other Jenkins server. We know how to setup a Jenkins job to run the pipeline even after several months...
     Also, it's easy for other pipeline script to call it.